src/components/Header.tsx

```diff
@@ -0,0 +1,49 @@
+import classNames from 'classnames';
+import React from 'react';
+
+interface Props {
+  isThereActiveTodo: boolean,
+  inputValue: string,
+  apiResponseReceived: boolean,
+  handleFormSubmit: (event: React.FormEvent<HTMLFormElement>) => Promise<void>,
+  updateTempTodo: (value: string) => void,
+  updateAllTodo: () => Promise<void>,
+}
+
+export const Header: React.FC<Props> = ({
+  isThereActiveTodo,
+  inputValue,
+  handleFormSubmit,
+  updateTempTodo,
+  apiResponseReceived,
+  updateAllTodo,
+}) => {
+  return (
+    <header className="todoapp__header">
+      <label htmlFor="nameInput">
+        <button
+          id="nameInput"
+          type="button"
+          className={classNames('todoapp__toggle-all', {
+            active: isThereActiveTodo,
+          })}
+          onClick={updateAllTodo}
+        >
+          {null}
```

Comment:
Empty content inside the 'button' element is not accessible. Add descriptive text or an 'aria-label' attribute for screen readers.